packages/components/src/Button/Button.css

```diff
@@ -248,54 +200,29 @@ a.button.disabled:hover {
 }
 
 .onlyIcon {
-  width: calc(var(--base-unit) * 2.25);
+  width: var(--button--size);
   padding: 0;
 }
 
-.onlyIcon.small {
```

Comment:
Because we're re-setting the button size variable on the `.small` and `.large` classes, we don't need to redefine te width values here